packages/app/src/routers/SynchronizationRouter/components/ConnectionsTable.tsx

```diff
@@ -85,6 +88,10 @@ export const ConnectionsTable = ({
                     accessToken
                   );
                   const runConnection = async () => {
+                    if (!synchContext.isAuthorized) {
+                      synchContext.login();
```

Comment:
`synchContext` needs to be added to the dep list

Reply:
nice catch. Updated